src/common/components/organisms/SearchAutocompleteInput.tsx

```
@@ -57,11 +60,16 @@ const SearchAutocompleteInputContent: React.FC<SearchAutocompleteInputProps> = (
     onSelect && onSelect();
   }, []);
 
+  const onSelectToken = useCallback((token: TokenResult) => {
+    router.push(`/t/${token.network}/${token.contractAddress}`);
+    onSelect && onSelect();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)